### PR TITLE
Hotfix/v2.0.1 (develop)

### PR DESCRIPTION
**Pull request by AlbertPG**
Closing #106 
Closing #114 
Closing #115

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/pethealth/TrGetAllWeights.java  4
- app/src/test/java/org/pesmypetcare/mypetcare/controllers/pethealth/TestTrGetAllWeights.java  1
         

Complexity decreasing per file
==============================
+ app/src/main/java/org/pesmypetcare/mypetcare/controllers/user/TrDeleteUser.java  -1
         

Clones removed
==============
+ app/src/main/java/org/pesmypetcare/mypetcare/activities/views/chart/statisticdata/WeeklyKilocaloriesData.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5581367&bid=18295417)

File: app/src/main/java/org/pesmypetcare/mypetcare/controllers/meals/TrDeleteMeal.java [hunk 1]
```diff
@@ -56,7 +56,8 @@ public void execute() throws NotPetOwnerException, ExecutionException, Interrupt
             throw new NotPetOwnerException();
         }
         mealManagerService.deleteMeal(user, pet, meal);
-        pet.deleteEvent(meal);
         googleCalendarService.deleteEvent(pet, meal);
+        System.out.println("Soy la transaccion");
```
Comment (xavier-campos-diaz) [inline app/src/main/java/org/pesmypetcare/mypetcare/controllers/meals/TrDeleteMeal.java]: I forgot to remove this print, you should remove it before merging this branch

File: app/src/main/java/org/pesmypetcare/mypetcare/features/pets/Pet.java [hunk 2]
```diff
@@ -461,10 +461,12 @@ public void addEvent(Event event) {
      * @param event The event to delete
      */
     public void deleteEvent(Event event) {
+        System.out.println("Soy el delete event");
         if (event instanceof Meals) {
             DateTime eventDate = event.getDateTime();
             double kcal = ((Meals) event).getKcal();
             try {
+                System.out.println("Soy el delete event y ademas soy un meal");
```
Comment (xavier-campos-diaz) [inline app/src/main/java/org/pesmypetcare/mypetcare/features/pets/Pet.java]: You should also remove those prints

**Review by xavier-campos-diaz — COMMENTED** [review]
You should fix some minor issues

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/test/java/org/pesmypetcare/mypetcare/controllers/pethealth/TestTrGetAllWeights.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/pethealth/TrGetAllWeights.java  4
         

Complexity decreasing per file
==============================
+ app/src/main/java/org/pesmypetcare/mypetcare/controllers/user/TrDeleteUser.java  -1
         

Clones removed
==============
+ app/src/main/java/org/pesmypetcare/mypetcare/activities/views/chart/statisticdata/WeeklyKilocaloriesData.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5581367&bid=18295417)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/pethealth/TrGetAllWeights.java  4
- app/src/test/java/org/pesmypetcare/mypetcare/controllers/pethealth/TestTrGetAllWeights.java  1
         

Complexity decreasing per file
==============================
+ app/src/main/java/org/pesmypetcare/mypetcare/controllers/user/TrDeleteUser.java  -1
         

Clones removed
==============
+ app/src/main/java/org/pesmypetcare/mypetcare/activities/views/chart/statisticdata/WeeklyKilocaloriesData.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5581367&bid=18295417)

**Review by xavier-campos-diaz — APPROVED** [review]

**Review by danicm47 — APPROVED** [review]

**Pull request merged**